web/src/test/manual/web/index.html

```diff
@@ -64,6 +64,7 @@ <h2><a href="./caps-lock-layer-3620/">Test Caps Lock Layer (#3620)</a></h2>
   <h2><a href="./start-of-sentence-3621/">Test Start of Sentence (#3621)</a></h2>
   <h2><a href="./start-of-sentence-3621/">Test start of sentence keyboard rules (#5963)</a></h2>
   <h2><a href="./issue6005/">Tests predictive text & other handling of rule matching when the final rule group does not match (#6005)</a></h2>
+  <h2><a href="./issue9469/">Test special characters rendering with keymanweb-osk.ttf (#9469)</a></h2>
```

Comment:
@mcdurdin - if I add a test page here, do I need to update something else so @bharanidharanj is able to get the [test artifact](https://build.palaso.org/repository/download/Keymanweb_TestPullRequests/403532:id/src/test/manual/web/)?

Reply:
I just pushed the build/test9469.js file. We'll see if that did it...

Reply:
Yep, the test page shows now